examples/debug/index.js

```diff
@@ -74,7 +74,18 @@ function buildExample(category, filename) {
     }));
 }
 
+if (!fs.existsSync(`${MAIN_DIR}/dist/build/playcanvas-observer.js`)) {
+    const playcanvasObserverText = fs.readFileSync(`${MAIN_DIR}/node_modules/@playcanvas/observer/index.js`);
+    const transformedPlaycanvasObserver = Babel.transform(playcanvasObserverText, {
+        retainLines: true,
+        filename: `playcanvas-observer.js`,
+        presets: ["env"]
+    }).code;
+    fs.writeFileSync(`${MAIN_DIR}/dist/build/playcanvas-observer.js`, transformedPlaycanvasObserver);
+}
```

Comment:
If you just deploy the latest Observer, Babel is now enabled and you don't need to do all this.

Reply:
Oh nice, thanks. I'll publish a new version of it now